### PR TITLE
Add a Darkmode toggle and Add my profile

### PR DESCRIPTION
**Pull request by jainpatel25**
please review it and can you add a hactoberfest tag?

### CODE REVIEW

File: pract/index.html [hunk 1]
```diff
@@ -0,0 +1,334 @@
+<!DOCTYPE html>
```
Comment (ShehrozIrfan) [inline pract/index.html:1]: Hey @jainpatel25 
What is `pract/` directory?
You need to add your code for html in `index.html`, styles in 'assets/css/style.css' and JavaScript in `assets/javascript/app.js`

File: pract/switch.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```
Comment (ShehrozIrfan) [inline pract/switch.html:1]: Hey @jainpatel25 
What is the purpose of this file?

File: pract/style1.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
+body {
```
Comment (ShehrozIrfan) [inline pract/style1.css:1]: @jainpatel25 
Do add your css in `assets/css/style.css` file

File: pract/script.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
+
```
Comment (ShehrozIrfan) [inline pract/script.js:1]: @jainpatel25 
Add this javascript in `assets/javascript/app.js`

**Comment by jainpatel25** [comment]
I will change it but do i need to pull another request for that?

**Comment by ShehrozIrfan** [comment]
> I will change it but do i need to pull another request for that?

@jainpatel25 
You can make changes and push it to the same branch. No need for new pull request.

**Comment by ShehrozIrfan** [comment]
@jainpatel25 
Once you're done, please do add image or small video to demonstrate how it is looking and working.

**Comment by jainpatel25** [comment]
I delete that pract file and made a PR please check it and

**Comment by ShehrozIrfan** [comment]
@jainpatel25 
It seems that the PR is not solving the issue?
If you want to add your profile then create separate PR for that with just your profile added. But make sure to pull the latest master first using:
` git pull origin master`
and then create a branch from master using:
`git checkout -b branch-name-here`
Add, commit and push your changes and then create a separate PR.

**Comment by jainpatel25** [comment]
Hey, but i added a dark mode toggler here so, it's working or not?

**Pull request closed without merging**